airflow/contrib/hooks/aws_sqs_hook.py

```diff
@@ -20,7 +20,7 @@
 
 import warnings
 
-from airflow.providers.amazon.aws.hooks.sqs import SQSHook  # noqa
+from airflow.providers.amazon.aws.hooks.sqs import SqsHook  # noqa
```

Comment:
This is not backward compatible. We are changing the class name.
So with the current code when user try to import SQSHook it won't work
Check reference of SESHook https://github.com/apache/airflow/pull/20367/files

It should

Reply:
Thanks. I understand that some of the changes aren't backward compatible but, this file has a deprecation warning here already. Do I just revert the changes here and create a deprecation warning for the `SQSHook` and make it backward compatible like this

```
class SQSHook(SqsHook):
    """
        This hook is deprecated.
        Please use :class:`airflow.providers.amazon.aws.hooks.sqs.SqsHook`.
        """

    def __init__(self, *args, **kwargs):
        warnings.warn(
            "This hook is deprecated. " "Please use :class:`airflow.providers.amazon.aws.hooks.sqs.SqsHook`.",
            DeprecationWarning,
            stacklevel=2,
        )
        super().__init__(*args, **kwargs)
```

Reply:
yes. The current implementation support deprecation where the class name was not change so it just add an import of where to find it but now -  the class name did change so we must address it. Yes the class you wrote should do the trick.